tests/dummy/public/test-forms/tables/form.ttl

```diff
@@ -0,0 +1,98 @@
+@prefix form: <http://lblod.data.gift/vocabularies/forms/> .
+@prefix sh: <http://www.w3.org/ns/shacl#>.
+@prefix mu: <http://mu.semte.ch/vocabularies/core/> .
+@prefix displayTypes: <http://lblod.data.gift/display-types/> .
+@prefix ext: <http://mu.semte.ch/vocabularies/ext/> .
+@prefix schema: <http://schema.org/>.
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>.
+@prefix nie: <http://www.semanticdesktop.org/ontologies/2007/01/19/nie#>.
+
+##########################################################
+# form
+##########################################################
+ext:form a form:Form, form:TopLevelForm ;
+    form:includes ext:administrativeUnitsL.
+
+##########################################################
+#  property-group
+##########################################################
+ext:mainPg a form:PropertyGroup;
+    sh:name "Bewerk betrokken lokale besturen" ;
+    sh:order 1 .
+
+##########################################################
+#  listing-scope: administrativeUnit
+##########################################################
+ext:administrativeUnitListingS a form:Scope;
+  sh:path ext:administrativeUnit.
+
+##########################################################
+#  Listing: Administrative unit
+##########################################################
+ext:administrativeUnitsL a form:Listing;
+  a form:ListingTable; #TODO
```

Comment:
At the moment this is the only change compared to the regular listing / sub form setup. Adding this class switches the display to the table version.

Is this the way to go or are there better options? We could also drop the form:Listing class and create whole new components instead of reusing things.

Reply:
Well the good thing here, for the backend, it won't change a thing. full transparent. 
The subform however, when in table mode, seems indeed also affected and worth a component on it's own. So perhaps double type here, and split table listing and table subform into really their own components no?

We need further discussion for sure.